src/configuration.ts

```diff
@@ -19,6 +19,13 @@ import * as fs from "fs";
 import * as path from "path";
 import * as findup from "findup-sync";
 
+import {arrayify, objectify} from "./utils";
+
+export interface IConfigFile {
```

Comment:
Agreed, the tricky thing here is we already have public APIs like
'DEFAULT_CONFIG' before that don't obey this convention.

On Fri, Mar 25, 2016, 12:04 PM Adi Dahiya <notifications@github.com> wrote:

> In src/configuration.ts
> <https://github.com/palantir/tslint/pull/1035#discussion_r57458084>:
>
> > @@ -19,6 +19,13 @@ import * as fs from "fs";
> >  import * as path from "path";
> >  import * as findup from "findup-sync";
> >
> > +import {arrayify, objectify} from "./utils";
> > +
> > +export interface IConfigFile {
>
> anything exported *is* part of the public API; let's just assume people
> will use anything they can find in our generated typings
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/tslint/pull/1035/files/feef9515728e2b920fb4a5b7d3bc9b569aa9ab6a#r57458084>
>